jwst/tests_nightly/general/miri/test_miri_steps_single.py

```diff
@@ -359,26 +359,182 @@ def test_miri_lrs_masterbg_user(self):
         input_sci.close()
 
 
-class TestMIRIMasterBackgroundMRS(BaseJWSTTest):
+@pytest.mark.bigdata
+class TestMIRIMasterBackgroundMRSDedicated(BaseJWSTTest):
+    input_loc = 'miri'
+    ref_loc = ['test_masterbackground', 'mrs', 'dedicated', 'truth']
+    test_dir = ['test_masterbackground', 'mrs', 'dedicated']
+
+    rtol = 0.000001
+
+    def test_miri_masterbg_mrs_dedicated(self):
+        """Run masterbackground step on MIRI MRS association"""
+        asn_file = self.get_data(*self.test_dir,
+                                  'miri_mrs_mbkg_0304_spec3_asn.json',
+                                  docopy=True)
```

Comment:
Since `docopy=True` always now, you don't need this in the call to `get_data`, nor in the subsequent 7 calls to the function.